app/views/layouts/_header.html.erb

```diff
@@ -9,7 +9,15 @@
         <span class="icon-bar"></span>
         <span class="icon-bar"></span>
       </button>
-      <%= link_to(image_tag("3sixd.png", :class => "navbar-brand", :alt => "Company logo"), root_path) %>
+      <% if Company.last.logo.present? %>
```

Comment:
@godfreym  , it was not put in the PR description, so there is no way I could have known it was not final implementation